### PR TITLE
Merged upstream master

### PR DESCRIPTION
**Pull request by Elad-Laufer**
Note that for the purpose of the merge I removed the custom sharpen

### CODE REVIEW

**Comment by alon-ne** [comment]
> Note that for the purpose of the merge I removed the custom sharpen

What do you mean, for the purpose of this merge? We don't want to remove custom sharpen, right? So where is it removed and where is it not?

File: vips/convolution.c [hunk 1]
```diff
@@ -4,6 +4,6 @@ int gaussian_blur_image(VipsImage *in, VipsImage **out, double sigma) {
 	return vips_gaussblur(in, out, sigma, NULL);
 }
 
-int sharpen_image(VipsImage *in, VipsImage **out, double sigma, double x1, double m2, VipsSharpenMode mode) {
-	return vips_sharpen(in, out, "sigma", sigma, "x1", x1, "m2", m2, "mode", mode, NULL);
+int sharpen_image(VipsImage *in, VipsImage **out, double sigma, double x1, double m2) {
```
Comment (alon-ne) [inline vips/convolution.c:7]: If we merge this without the sharpen mode parameter, it will break IM

**Review by alon-ne — CHANGES_REQUESTED** [review]

File: vips/convolution.c (hunk 1 above)
Comment (Elad-Laufer) [reply vips/convolution.c:7]: I'll reinstate it in master after merge,

Comment (alon-ne) [reply vips/convolution.c:7]: Let's reinstate it here and then merge, that's the right way to work

**Pull request closed without merging**